presto-memory/src/test/java/com/facebook/presto/plugin/memory/TestMemorySmoke.java

```diff
@@ -125,6 +126,21 @@ public void selectColumnsSubset()
         assertThatQueryReturnsSameValueAs("SELECT nationkey, regionkey FROM nation ORDER BY nationkey", "SELECT nationkey, regionkey FROM tpch.tiny.nation ORDER BY nationkey");
     }
 
+    @Test
+    public void createTableInNonDefaultSchema()
```

Comment:
I don't understand why, isn't it just an additional noise? Besides, usually, but not always. There are a lot of tests that do not follow this convention and I have copied this convention from `TestMemorySmoke`/`TestBlackHoleSmoke` which on the other hand are based on `TestHiveIntegrationSmoke`.